src/qibosoq/__main__.py

```diff
@@ -0,0 +1,14 @@
+from socketserver import TCPServer
+
+from qick import QickSoc
+
+from qibosoq.rfsoc_server import MyTCPHandler
+
+HOST = "192.168.0.72"  # Server address
+PORT = 6000  # Port to listen on
+
+TCPServer.allow_reuse_address = True
+with TCPServer((HOST, PORT), MyTCPHandler) as server:
+    print("Server Listening")
+
+    server.serve_forever()
```

Comment:
Wrap this in a function:
```suggestion
def serve(host, port):
    TCPServer.allow_reuse_address = True
    with TCPServer((host, port), MyTCPHandler) as server:
        print("Server Listening")

        server.serve_forever()
        
serve(HOST, PORT)
```

It will be easier to upgrade.

Reply:
P.S.: the `serve()` function can also be located back in the `rfsoc_server` module :)